herddb-jdbc/src/main/java/herddb/jdbc/Driver.java

```diff
@@ -120,4 +124,19 @@ public synchronized void close() {
         datasources.clear();
     }
 
+    /**
+     * Closes drive embedded datasources by their connection url
+     */
+    public synchronized void closeDatasources(String url) {
```

Comment:
This is not cleaer to me.
The best way is to call "unregister" on the JDBC driver, this will call "close()" which will close all the datasources retained by the driver itself.

```
List<Driver> dd = Collections.list(DriverManager.getDrivers());
            dd.stream().forEach((d) -> {
                try {
                    System.out.println("[JDBC] unregister driver " + d.getClass());
                    DriverManager.deregisterDriver(d);
                } catch (Throwable t) {
                    System.out.println("[JDBC] unable to unregister driver " + d.getClass() + ": " + t);
                }
            });
```